@ndlib/gatsby-theme-marble/src/components/Layout/index.js

```diff
@@ -68,4 +69,4 @@ Layout.defaultProps = {
   requireLogin: false,
 }
 
-export default Layout
+export default withI18nTranslation(Layout)
```

Comment:
We wrap the Layout in the i18next context provider component so that all child components can attach to the same context.